PresentData/ETW/Microsoft_Windows_DxgKrnl.h

```diff
@@ -155,6 +156,11 @@ struct Blit_Info_Struct {
     uint32_t    SubRectCount;
 };
 
+template<typename PointerT>
+struct Blit_Cancel_Struct {

```

Comment:
I'm having trouble finding what these are actually used for. Do we need one here?

Reply:
We don't actually read any payload from this event, so this indeed goes unused. I can get rid of it.

Reply:
No they're generated by etw_list tool but not used by any PresentMon code.  That said, they can have use as documentation e.g., when re-running etw_list later it can be useful to see diffs in properties/versions etc.

Reply:
Is this one interesting for that case, given we don't utilize any of the data? Happy to add it back if you'd like it.

Reply:
Not a big deal... either way :)